lib/compute/network.js

```diff
@@ -215,6 +221,137 @@ Network.prototype.createFirewall = function(name, config, callback) {
 };
 
 /**
+ * Create a subnetwork in this network.
+ *
+ * @resource [Subnetwork Resource]{@link https://cloud.google.com/compute/docs/reference/v1/subnetworks#resource}
+ * @resource [Subnetwork: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/subnetworks/insert}
+ *
+ * @param {string} name - Name of the subnetwork.
+ * @param {object} config - See a
+ *     [Subnetwork resource](https://cloud.google.com/compute/docs/reference/v1/subnetworks#resource).
+ * @param {string=} config.description - An optional description of this
+ *     resource. Provide this property when you create the resource.
+ * @param {string} config.ipCidrRange - The range of internal addresses that
```

Comment:
Here's an example from `createNetwork`: https://github.com/GoogleCloudPlatform/gcloud-node/blob/02928725218caf4febf49911ae3ec68d0ee581e8/lib/compute/index.js#L358 

The URL to the CIDR wiki may be helpful as well as `(Alias for `config.ipCidrRange`)`.